apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -12,11 +12,24 @@
 {{#inline}}<span class="form-inline d-inline-block ml-2">{{/inline}}
     <span id="pl-number-input-{{uuid}}" class="input-group pl-number-input">
         {{#label}}
+        <label for="pl-number-input-textfield-{{uuid}}" hidden>{{accessibility_description}}</label>
```

Comment:
I think this should be an `arialabel` on the input tag? cc @nwalters512

Reply:
It doesn't seem like this label is needed? I don't even think the `accessibility_description` is being set anywhere.